app/views/ExposureHistory/ExposureDatumDetail.tsx

```diff
@@ -108,28 +96,6 @@ const NoKnownExposureDetail = ({
   );
 };
 
-interface UnknownExposureDetailProps {
```

Comment:
It turns out that we dont get info back to determine if the user was or was not tracking exposure notifications on a day. So we cant make the determination that we have an Unknown ExposureDatum.  We just have, Possible and NoKnown.

Reply:
> we dont get info back to determine if the user was or was not tracking exposure notifications on a day

My understanding is this is inferred from being in an app state of not being active? e.g. on GPS if location services were off, or if locations are missing for a day, that's different than NoKnown

Reply:
yeah for GPS, it might make sense to have this concept, so we could leave this in, if you think itll be useful, but for BT we simply dont have that data. All we have is if there was an exposure or not.  At least at this point we may have a way to get that info down the line.

Reply:
Do you know where this copy originally came from? If this was just added specifically for BT I'm good with removing it

side note: the strings in this file should be localized?

Reply:
I initially put it in as it was in the figma, I did chat with Emma about it a bit but we werent sure. Ali would have more context on the why. My guess is that it was assumed to be an easy thing to do, and now that we've worked through it, its actually a bigger ask, so it would be worth revisiting if its a useful feature to the user. Either way, i think it belongs in a separate commit.

Reply:
> side note: the strings in this file should be localized? out of scope for this PR but in general

yes totally, was skipped for sake of the demo, we cut a card to capture the work.